xlsynth/src/dslx.rs

```diff
@@ -49,12 +49,14 @@ impl ImportData {
             std::ffi::CString::new(dslx_stdlib_path.to_str().unwrap()).unwrap();
         let dslx_stdlib_path_c_ptr = dslx_stdlib_path_c_str.as_ptr();
 
-        let mut additional_search_paths_ptr: Vec<*const std::os::raw::c_char> = vec![];
+        let mut additional_search_paths_cstring: Vec<std::ffi::CString> = vec![];
         for path in additional_search_paths {
-            let path_c_str = std::ffi::CString::new(path.to_str().unwrap()).unwrap();
-            additional_search_paths_ptr.push(path_c_str.as_ptr());
+            let path_c_string = std::ffi::CString::new(path.to_str().unwrap()).unwrap();
```

Comment:
any specific reason you don't use `map` here too? (is that because the intention here is more to build something new rather than transforming it?)

Reply:
Good point, fixed.